src/include/cursor.h

```diff
@@ -54,20 +54,22 @@ struct __wt_cursor_backup {
 
     WT_CURSOR *incr_cursor; /* File cursor */
 
-    bool incr_init;       /* Cursor traversal initialized */
     WT_ITEM bitstring;    /* List of modified blocks */
     uint64_t nbits;       /* Number of bits in bitstring */
     uint64_t offset;      /* Zero bit offset in bitstring */
     uint64_t bit_offset;  /* Current offset */
     uint64_t granularity; /* Length, transfer size */
 
 /* AUTOMATIC FLAG VALUE GENERATION START */
-#define WT_CURBACKUP_DUP 0x01u        /* Duplicated backup cursor */
-#define WT_CURBACKUP_FORCE_FULL 0x02u /* Force full file copy for this cursor */
-#define WT_CURBACKUP_FORCE_STOP 0x04u /* Force stop incremental backup */
-#define WT_CURBACKUP_INCR 0x08u       /* Incremental backup cursor */
-#define WT_CURBACKUP_LOCKER 0x10u     /* Hot-backup started */
-                                      /* AUTOMATIC FLAG VALUE GENERATION STOP */
+#define WT_CURBACKUP_CKPT_FAKE 0x01u   /* Object has fake checkpoint */
+#define WT_CURBACKUP_DUP 0x02u         /* Duplicated backup cursor */
+#define WT_CURBACKUP_FORCE_FULL 0x04u  /* Force full file copy for this cursor */
+#define WT_CURBACKUP_FORCE_STOP 0x08u  /* Force stop incremental backup */
+#define WT_CURBACKUP_HAS_CB_INFO 0x10u /* Object has checkpoint backup info */
+#define WT_CURBACKUP_INCR 0x20u        /* Incremental backup cursor */
+#define WT_CURBACKUP_INCR_INIT 0x40u   /* Cursor traversal initialized */
+#define WT_CURBACKUP_LOCKER 0x80u      /* Hot-backup started */
```

Comment:
I know it fits into the 8 bits, but we've had some problems with flags creeping over boundaries recently, so I'm paranoid. Could you increase the size of this to a `uint32_t` - we aren't particularly size sensitive in this structure.

Reply:
Fixed.